### PR TITLE
Update documentation for Ember 3.x tests

### PR DESCRIPTION
**Pull request by happycollision**
I know this is probably not enough, but it is a start, and would have been helpful to me.

### CODE REVIEW

File: docs/v0.4.x/installation.md [hunk 1]
```diff
@@ -12,3 +12,22 @@ ember install ember-cli-mirage
...
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';  // <-- New
+
+module('Acceptance | mirage', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks); // <-- New
+
+  test('server exists', async function(assert) {
+    assert.ok(server);
```
Comment (nightire) [inline docs/v0.4.x/installation.md]: is `this.server` a more recommended way of accessing?

Comment (happycollision) [reply docs/v0.4.x/installation.md]: I have no idea.

**Comment by omairvaiyani** [comment]
```import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';``` results in ```"Module not found"``` error.

I'm on Ember 3.0.0

**Comment by givanse** [comment]
I just upgraded an app to Ember CLI 3 and my acceptance tests still work fine. I didn't need to add what is shown here.

However, my integration/unit tests are definitely broken. [Those were manually starting Mirage](http://www.ember-cli-mirage.com/docs/v0.4.x/manually-starting-mirage/). (still figuring it out, will edit this comment if I find the way)

**Comment by chilicoder** [comment]
I needed this docs to run mirage in ember 3.1 acceptance tests

**Comment by happycollision** [comment]
@givanse I think that Mirage, like many addons, is supporting both the old test style and the new one. If you generate new tests, they may be in the new style, but your old ones will continue to work. If it isn't too much work to update your unit and integration tests, then you should be able to use `setupMirage(hooks)`. I use mirage in a few unit tests (I know, I know 🤷‍♂️) and several integration tests. I do not know about how to keep the old version working with manual starting.

@omairvaiyani Have you also made sure that you have also updated Mirage to the latest version?

I think I'll be closing this because it looks like @samselikoff may have added [necessary docs](https://github.com/samselikoff/ember-cli-mirage/commit/4b6176cec1f44b6902c445f34fa051ef2e6f5493).

**Comment by omairvaiyani** [comment]
@happycollision I managed to get it working by following the acceptance test setup guidance on Ember Docs

**Pull request closed without merging**